homeassistant/components/fronius/config_flow.py

```diff
@@ -104,6 +115,45 @@ async def async_step_import(self, conf: dict) -> FlowResult:
         """Import a configuration from config.yaml."""
         return await self.async_step_user(user_input={CONF_HOST: conf[CONF_RESOURCE]})
 
+    async def async_step_dhcp(self, discovery_info: DhcpServiceInfo) -> FlowResult:
+        """Handle a flow initiated by the DHCP client."""
+        for entry in self._async_current_entries(include_ignore=False):
+            if entry.data[CONF_HOST].lstrip("http://").rstrip("/").lower() in (
+                discovery_info.ip,
+                discovery_info.hostname,
+            ):
+                return self.async_abort(reason="already_configured")
+        # Symo Datalogger devices need up to 1 minute at boot from DHCP request
+        # to respond to API requests (connection refused until then)
+        await asyncio.sleep(60)
```

Comment:
Maybe you could create a constant here `DHCP_REQUEST_DELAY`, then you could patch that instead of patching `asyncio.sleep`

Reply:
Good idea! 👍